README.md

```diff
@@ -115,6 +115,7 @@ Rule Name   | Description | Since
 `no-unnecessary-local-variable` | Do not declare a variable only to return it from the function on the next line. It is always less code to simply return the expression that initializes the variable. | 2.0.4
 `no-unnecessary-override`       | Do not write a method that only calls super() on the parent method with the same arguments. You can safely remove methods like this and Javascript will correctly dispatch the method to the parent object. | 2.0.4
 `no-unnecessary-semicolons`     | Remove unnecessary semicolons | 0.0.1
+`no-unsupported-browser-code`   | Avoid writing browser-specific code for unsupported browser versions. Browser versions are specified in the rule configuration options, eg: \[true, \[ "IE 11", "Firefox > 40", "Chrome >= 45" \] \]. Browser-specific blocks of code can then be designated with a single-line comment, like so: `// Browser specific: IE 10`, or with a jsdoc like this: `@browserspecific chrome 40`. As you drop support for browsers, you can modify your supported browsers in the config to have the linter throw errors | 2.0.9
```

Comment:
Nit: might look prettier if you remove the last sentence and wrapped the first code sample with code \`s too.